SPECS/pcre/pcre.spec

```diff
@@ -93,31 +101,45 @@ make %{?_smp_mflags} check
 -   Update Source0 with valid URL.
 -   Remove sha1 macro.
 -   License verified.
+
 *   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 8.42-2
```

Comment:
Could you also update the spacing and add the hyphens before the version number?